From 460b3d79a31231b43f91ab8595f2ee79fa9c3d56 Mon Sep 17 00:00:00 2001
From: Giuseppe Scrivano <gscrivan@redhat.com>
Date: Tue, 15 Apr 2014 16:22:33 +0200
Subject: [PATCH] virtinst: display the domain for PCI devices

Closes: https://bugzilla.redhat.com/show_bug.cgi?id=1085499

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>
(cherry picked from commit e980d9c737284a5e098a35ea6d8a17155aaf5ce9)

Conflicts:
	virtinst/nodedev.py
---
 virtinst/NodeDeviceParser.py | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/virtinst/NodeDeviceParser.py b/virtinst/NodeDeviceParser.py
index c6c8c26..3f595d0 100644
--- a/virtinst/NodeDeviceParser.py
+++ b/virtinst/NodeDeviceParser.py
@@ -197,9 +197,10 @@ class PCIDevice(NodeDevice):
             child = child.next
 
     def pretty_name(self, child_dev=None):
-        devstr = "%.2X:%.2X:%X" % (int(self.bus),
-                                   int(self.slot),
-                                   int(self.function))
+        devstr = "%.4X:%.2X:%.2X:%X" % (int(self.domain),
+                                        int(self.bus),
+                                        int(self.slot),
+                                        int(self.function))
         if child_dev:
             desc = "%s %s (%s)" % (devstr, child_dev.pretty_name(),
                                    str(self.product_name))
-- 
1.9.0

